test/integration/failing_test_heartbeat.sh

```diff
@@ -5,25 +5,28 @@ set -e
 CPODIR=/tmp/___cpo_integ_test
 rm -rf ${CPODIR}
 EPS="tcp+k2rpc://0.0.0.0:10000"
+NUMCORES=`nproc`
+# core on which to run the TSO poller thread. Pick 4 if we have that many, or the highest-available otherwise
+TSO_POLLER_CORE=$(( 5 > $NUMCORES ? $NUMCORES-1 : 4 ))
 
 PERSISTENCE=tcp+k2rpc://0.0.0.0:12001
 CPO=tcp+k2rpc://0.0.0.0:9000
 TSO=tcp+k2rpc://0.0.0.0:13000
 COMMON_ARGS="--poll-mode --thread-affinity false --enable_tx_checksum true"
 
-# start CPO on 1 cores
+# start CPO
 ./build/src/k2/cmd/controlPlaneOracle/cpo_main -c1 --tcp_endpoints ${CPO} --data_dir ${CPODIR} --enable_tx_checksum true --reactor-backend epoll --thread-affinity false --prometheus_port 63000 --assignment_timeout=1s --heartbeat_deadline=1s &
 cpo_child_pid=$!
 
-# start tso on 2 cores
-./build/src/k2/cmd/tso/tso -c2 --tcp_endpoints ${TSO} 13001 --enable_tx_checksum true --reactor-backend epoll --prometheus_port 63003 --thread-affinity false &
+# start tso
+./build/src/k2/cmd/tso/tso -c1 --tcp_endpoints ${TSO} --enable_tx_checksum true --reactor-backend epoll --prometheus_port 63003 --thread-affinity false --tso.error_bound=100us --tso.clock_poller_cpu=${TSO_POLLER_CORE} &
```

Comment:
Is the 1 core, i.e., -c1, only for the poller? how do we specify the number of workers on different cores?